src/autowiring/test/PostConstructTest.cpp

```diff
@@ -427,3 +414,19 @@ TEST_F(PostConstructTest, CorrectContextAssignment) {
 
   ASSERT_EQ(child, currentCtxt) << "Current context was not correctly assigned in a post-satisfied NotifyWhenAutowired handler";
 }
+
+namespace {
+  class ThrowsInAutoInit {
+  public:
+    void AutoInit(void) {
+      throw std::runtime_error("Failure!");
```

Comment:
Consider a message that is a unique string users can locate more easily.  E.g. "AutoInit failure"